static/sass/_pattern_cube-hero.scss

```diff
@@ -0,0 +1,16 @@
+@mixin ubuntu-p-cube-hero {
+  .p-cube-hero {
+    .p-cube-hero--content {
+      margin-top: 2rem;
+    }
+
+    @media (min-width: $breakpoint-large) {
+      padding-bottom: 1.5rem !important;
```

Comment:
:scream: Could this be achieved with a higher specificity instead of `!important`?

Reply:
I knew it felt dirty using `!important` :laughing: 
I removed it and increased the specificity by making the selector `.p-strip.p-cube-hero`.